share/man/man7/hier.7

```diff
@@ -65,14 +68,17 @@ and
 .Pp
 .Bl -tag -width "overlays/" -compact
 .It Pa overlays/
-compiled flattened device tree (FDT) overlays; these can be applied using the
+compiled
+.Xr fdt 4
+overlays; see
 .Va fdt_overlays
-setting in
+in
 .Xr loader.conf 5
 .El
 .Pp
 .It Pa efi/
-mount point for EFI System Partition (ESP) on UEFI systems
+.Xr UEFI 8
```

Comment:
```suggestion
.Xr uefi 8
```

We don't install a capitalized symlink, unfortunately.

Reply:
Thanks for the review and tutelage guys. I originally omitted the article so it would fit on one line, mistake. Next forcepush will read `The uefi(8)...`

I have drafts for uefi(8) and boot(8) since August that aren't even close.